vignettes/articles/layouts.Rmd

```diff
@@ -0,0 +1,245 @@
+---
+title: "Sample layouts"
```

Comment:
@apreshill Looks like the same content as in https://rmarkdown.rstudio.com/flexdashboard/layouts.html where the source is https://github.com/rstudio/rmarkdown/blob/gh-pages/flexdashboard/layouts.Rmd

Is this the case ? Should we try to avoid duplicating both source and have this article for pkgdown website "be sourced" on the one from the flexdashboard demo website ? 

I have an idea to do that.

Reply:
Hi @cderv - this site will replace https://rmarkdown.rstudio.com/flexdashboard/

Reply:
Oh ok ! In that case, it makes sense to move everything as articles in the pkgdown website! 😄